.github/workflows/ci-schedule-imagescanning.yaml

```diff
@@ -0,0 +1,79 @@
+name: image-scanning-on-schedule
+on:
+  schedule:
+    # Run this workflow "At 00:00 UTC on Sunday"
+    - cron: '0 0 * * 0'
+
+jobs:
+  use-trivy-to-scan-image:
+    name: image scannning
+    # prevent job running from forked repository
+    if: ${{ github.repository == 'karmada-io/karmada' }}
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      matrix:
+        karmada-version: [ release-1.6, release-1.7, release-1.8 ]

```

Comment:
You can add some comments here to remind us to make timely modifications after the subsequent version is released.

Reply:
nice suggestion! done~